advanced/button-interrupt/exercise/src/main.rs

```diff
@@ -57,12 +53,8 @@ fn main() -> anyhow::Result<()> {
 
             // 8. Receive the event from the queue.
             // let res = ...;
-            
-            // If the event has the value 0, nothing happens. if it has a different value, the button was pressed. 
-            match res {
-                1 => println!("button pressed!"),
```

Comment:
I left the match intentionally there so the focus is on the c bindings and not on writing Rust. It would be good to revert the numbering.